Tasks/BashV3/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -25,5 +25,5 @@
   "loc.messages.JS_Stderr": "Bash wrote one or more lines to the standard error stream.",
   "loc.messages.JS_TranslatePathFailed": "Unable to translate the path '%s' to the Linux file system.",
   "loc.messages.JS_BashEnvAlreadyDefined": "The BASH_ENV environment variable has already been set to a '%s', the task will override it with '%s'",
-  "loc.messages.SanitizerOutput": "Sanitizer changed input arguments. Output from sanitizer: '%s'"
+  "loc.messages.ScriptArgsSanitized": "Detected characters in arguments that may not be executed correctly by the shell. Please escape special characters using backslash (\\). More information is available here: <https://aka.ms/ado/75787>"
```

Comment:
Does the link in the < > get rendered correctly?

Reply:
fixed